lib/containers/accumulators/review-comments-accumulator.js

```diff
@@ -67,6 +67,7 @@ export default createPaginationContainer(BareReviewCommentsAccumulator, {
             author {
               avatarUrl
               login
+              url
```

Comment:
Oh interesting, we were just URL-hacking this in before? Yeah, much cleaner to get it from GraphQL 👍